app/lib/theme/ReadOnlyAdditionalFundingSourcesArrayFieldTemplate.tsx

```diff
@@ -0,0 +1,41 @@
+import { ArrayFieldTemplateProps } from "@rjsf/core";
```

Comment:
This should probably also live in our `components` folder somewhere near the other one